Makefile

```diff
@@ -14,7 +14,7 @@ build:
 	go build -v -ldflags "$(LDFLAGS)" -o $(BUILD_DIR)/$(NAME) ./cmd/$(NAME)
 
 package: build
-	zip -j $(BUILD_DIR)/$(NAME)-$(VERSION)-$(OS)-$(ARCH).zip $(BUILD_DIR)/$(NAME)
+	tar -czvf $(BUILD_DIR)/$(NAME)-$(VERSION)-$(OS)-$(ARCH).tar.gz $(BUILD_DIR)/$(NAME)
```

Comment:
Why not create both?

Reply:
I will answer with a question - why create both for linux package?